chopper/test/base_test.dart

```diff
@@ -104,28 +104,6 @@ void main() {
       httpClient.close();
     });
 
-    test('GET with query params, null value', () async {
-      final httpClient = MockClient((request) async {
```

Comment:
We can keep that test
and change the function call  `service.getQueryTest(def: null);` with an other one which can be nullable


=> `service.getQueryTest(number: null);`

Reply:
Done!

Reply:
Done!